packages/vscode-extension/src/utils/commonUtils.ts

```diff
@@ -105,7 +105,8 @@ export function anonymizeFilePaths(stack?: string): string {
   }
 
   const nodeModulesRegex = /^[\\\/]?(node_modules|node_modules\.asar)[\\\/]/;
-  const fileRegex = /(file:\/\/)?([a-zA-Z]:(\\\\|\\|\/)|(\\\\|\\|\/))?([\w-\._]+(\\\\|\\|\/))+[\w-\._]*/g;
+  const fileRegex =
+    /(file:\/\/)?([a-zA-Z]:(\\\\|\\|\/)|(\\\\|\\|\/))?([\w-\._]+(\\\\|\\|\/))+[\w-\._]*/g;
```

Comment:
shall we add some ut for this regex?

Reply:
This is telemetry old code. I think this will be added when necessary.